src/routes/_pages/settings/wellness.html

```diff
@@ -33,14 +33,19 @@ <h2>{intl.metrics}</h2>
     </label>
   </form>
 
-  <h2>{intl.notifications}</h2>
+  <h2>{intl.immediacy}</h2>
 
   <form class="ui-settings">
     <label class="setting-group">
       <input type="checkbox" id="choice-disable-unread-notification-counts"
              bind:checked="$disableNotificationBadge" on:change="onChange(event)">
       {intl.hideUnread}
     </label>
+    <label class="settings-group">
+      <input type="checkbox" id="choice-disable-relative-timestamps"
+             bind:checked="$disableRelativeTimestamps" on:change="onChange(event)">
+      {intl.showAbsoluteTimestamps}
+    </label>
```

Comment:
`flushChangesToCheckAll` in this file would also need to be updated.

Reply:
Done!